### PR TITLE
Final features

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
### What?

I created the structure for a to-do list that helps people manage their activities.

### Why?

This is to help them manage their activities and also help them plan their days easily.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -40,29 +41,25 @@ submit.addEventListener('click', (event) => {
...
 
 // event listener for clear completed button
 clearCompletedButton.addEventListener('click', (event) => {
   event.preventDefault();
   const completedTasks = document.querySelectorAll('.completed');
   completedTasks.forEach((task) => {
-    task.remove();
+    task.parentElement.remove();
   });
 });
```
Comment (rloterh) [inline src/index.js]: - [ ] This activity requires that you use `Array.filter()` method alongside with the _Clear all completed_ button in removing all completed tasks, which is not implemented in your project. Kindly ensure to improve upon the same.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Abubakar, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Tasks are saved to local storage correctly
- Local storage updates correctly on users' action


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. 
- I think you should replace the sub heading text Author1 with your full name in your ReadMe.
Implementing the above change(s) will make your ReadMe appear more professional.
<br>
Cheers and Happy coding!👏 💻 🍷
<br><br>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Haywayaheadshot) [reply src/index.js]: @rloterh Thank you for reviewing my project. I will work on the changes asap

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Haywayaheadshot, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Haywayaheadshot** [comment]
@Mihndim2020 Thank you so much for reviewing my code. Have a great weekend.

**Pull request merged**